alchemy/src/cloudflare/bucket.ts

```diff
@@ -202,88 +225,119 @@ export function isBucket(resource: Resource): resource is R2Bucket {
  *
  * @see https://developers.cloudflare.com/r2/buckets/
  */
-export const R2Bucket = Resource(
+export async function R2Bucket(
+  id: string,
+  props: BucketProps = {},
+): Promise<R2Bucket> {
+  return await _R2Bucket(id, {
+    ...props,
+    dev: {
+      ...(props.dev ?? {}),
+      force: Scope.current.local,
+    },
+  });
+}
```

Comment:
As we duplicate this, I wonder if we can solve it more generally.

e.g.
```ts
export interface ResourceProps {
  dev?: {
    force?: boolean;
  }
}
```

And then have `Resource` do this automatically.

Problem is I am not sure how to identify when NOT to do it. Unless we support something like:
```ts
export const MyResource = Resource("foo::bar", { local: true }, async function(..) { .. });
```

Reply:
Let's not solve it in this PR, but think about it.